calculator/main.py

```diff
@@ -26,7 +32,26 @@ def perform_operation(calc, operation, params):
     :param params: Tuple containing the list of nums to operate with.
                    ie: (1, 2, 3, 4.5, -2)
     """
-    pass
+    if not params or not isinstance(params, tuple):
```

Comment:
Don't need to check `not params` here, params is a required parameter of `perform_operation` (as in it doesn't have a default value) so all this is doing is essentially check if it's either `None` or `False`, which the isinstance part of the if statement handles.